annlite/index.py

```diff
@@ -747,13 +748,13 @@ def _backup_index_to_remote(self, target_name: str, token: str):
                 cell_id=cell_id,
             )
 
-        # upload meta table
-        uploader.upload_file(
-            Path(self.data_path) / 'metas.db',
-            target_name=target_name,
-            type='meta_table',
-            cell_id='all',
-        )
+        # # upload meta table
```

Comment:
why not upload meta table?

Reply:
good catch, this is an error I made. I forgot to uncomment them.

Reply:
oh, i see, the uploading brokes our tests. https://github.com/jina-ai/annlite/actions/runs/4507514031/jobs/7935366275?pr=222 @jemmyshin Do you have any clues about it?